homeassistant/components/vesync/light.py

```diff
@@ -0,0 +1,85 @@
+"""Support for VeSync dimmers."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS,
+    LightEntity,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .common import VeSyncDevice
+from .const import DOMAIN, VS_DISCOVERY, VS_DISPATCHERS, VS_LIGHTS
+
+_LOGGER = logging.getLogger(__name__)
+
+DEV_TYPE_TO_HA = {
+    "ESD16": "light",
+    "ESWD16": "light",
+}
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up lights."""
+
+    async def async_discover(devices):
+        """Add new devices to platform."""
+        _async_setup_entities(devices, async_add_entities)
+
+    disp = async_dispatcher_connect(
+        hass, VS_DISCOVERY.format(VS_LIGHTS), async_discover
+    )
+    hass.data[DOMAIN][VS_DISPATCHERS].append(disp)
+
+    _async_setup_entities(hass.data[DOMAIN][VS_LIGHTS], async_add_entities)
+    return True
+
+
+@callback
+def _async_setup_entities(devices, async_add_entities):
+    """Check if device is online and add entity."""
+    dev_list = []
+    for dev in devices:
+        if DEV_TYPE_TO_HA.get(dev.device_type) == "light":
+            dev_list.append(VeSyncDimmerHA(dev))
+        else:
+            _LOGGER.warning(
```

Comment:
```suggestion
            _LOGGER.debug(
```

Sorry I missed this on the first go:

We should have a docs PR to tell users which devices are supported to be consistant with the existing docs.